bin/tessel-2.js

```diff
@@ -21,11 +21,16 @@ function makeCommand(commandName) {
     })
     .option('lan', {
       flag: true,
-      help: 'Use LAN connection'
+      help: 'Use only a LAN connection'
     })
     .option('usb', {
       flag: true,
-      help: 'Use USB connection'
+      help: 'Use only a USB connection'
+    })
+    .option('lanPrefer', {
```

Comment:
This is just nit-picky but I think cli programs should avoid camel cased parameters